### PR TITLE
Color Mode Button/Notification UI Changed

### LINKED ISSUE #651: Feature Request — Dark Mode Button
**Job**
The user can click the moon instead of the button next to the moon to activate the dark mode.

**Possible Solution**
Delete the button from the interface and turn the moon into a button. Allow the moon to shine when dark mode is activated, and dim the moon when it is deactivated.

**Context**
Users can switch between dark and light mode with one clear button.

### PR DESCRIPTION
**Pull request by mlammon11**
- Changed the notification button and moon to a dropdown list so that it's more intuitive to the user
- Added brief text in front so the user instantly knows the function of the list
- Moved toast notification to the top-center of the screen so that it doesn't interfere with switching between themes

**Issue**
> closes #651

**Database Changes/Migrations**

Nothing changed here 

**Test Modifications**

Nothing changed here 

**Test Procedure**
- Use the new switch to change from light/dark mode
- Double click moon to see the notification moved to the top-middle of the screen, allowing you to adjust your settings without notification in your way

**Photos**
Old
image

New
image
image

Old
image

New 
image

**Additional Info**

### CODE REVIEW

**Review by dorian451 — CHANGES_REQUESTED** [review]
looks good so far,
can you make the moon more obviously clickable

**Review by jonha1 — APPROVED** [review]
looks awesome, improved user experience

File: src/web/src/components/Header.vue [hunk 1]
```diff
@@ -53,14 +53,12 @@
...
-        <b-nav-form id="darkmode-toggle-form" class="mr-md-2">
-          <b-form-checkbox
-            :checked="$store.state.darkMode"
-            :indeterminate="followDevice"
-            @change="toggle_style()"
-          ></b-form-checkbox>
-          <font-awesome-icon @dblclick="toggle_default()" icon="moon" />
-        </b-nav-form>
+
+        <b-nav-dropdown text="Color Mode" style="padding-right: 5px">
```
Comment (dorian451) [inline src/web/src/components/Header.vue:57]: can you make it so that the currently selected mode is highlighted or something

**Review by dorian451 — CHANGES_REQUESTED** [review]

**Review by zhengw5 — APPROVED** [review]
Function works just as the coder says. I am not entirely sure if I like the feature better but it is more intuitive.

**Review by dorian451 — CHANGES_REQUESTED** [review]
it doesn't follow device theme after the initial load, and doesn't allow user to change theme until after they select "follow device theme"

**Review by dorian451 — APPROVED** [review]
finally works, yay!

**Review by jonha1 — APPROVED** [review]
looks great

**Pull request merged**